scripts/check_increasing_levels.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+# A script to check that tariff levels are strictly increasing
+import os
+import yaml
+
+if __name__ == "__main__":
+    for filename in os.listdir("tariffer"):
+        if filename.endswith(".yml"):
+            with open(os.path.join("tariffer", filename), "r") as f:
+                data = yaml.safe_load(f)
+
+            for t in data["tariffer"]:
+                valid_from = t["gyldig_fra"]
+                levels = t["fastledd"]["terskler"]
+
+                levels.sort(key=lambda x: x["terskel"])
+
+                for i in range(len(levels) - 1):
+                    level = levels[i]["terskel"]
+                    next_level = levels[i + 1]["terskel"]
+
+                    price = levels[i]["pris"]
+                    next_price = levels[i + 1]["pris"]
+
+                    if next_level <= level:
+                        print(
+                            f"Level {next_level} in {filename} - {valid_from} is lower or equal to level {level}"
+                        )
+
+                    if next_price <= price:
```

Comment:
Enig i at terskel/level bør være strengt økende, men trenger vi sjekk på om prisen endres? 
Så kan jeg være enig i at det føles unødvendig med ny terskel dersom prisen er den samme

```suggestion
                    if next_price < price:
```

Reply:
Enig. Det holder å sjekke at prisen øker monotont. Tidligere har vi hatt feil hvor tersklene har blitt byttet om eller at vi plutselig har mndpris midt inni. Det oppdager vi fortsatt.